README.md

```diff
@@ -7,43 +7,40 @@
+Some features are still under heavy development, but existing features should work fine.
 
-If got any problem, you can just ask me in gmail(andy840119@gmail.com) or [discord channel](https://discord.gg/ga2xZXk)(will reply faster) instead of costing too much time in solving problem.
+If you run into any problems, you can shoot me an email (andy840119@gmail.com) or send me a message on [discord](https://discord.gg/ga2xZXk). I will typically reply faster on discord.
 
-And feel free to report bugs if found.
+And feel free to report any bugs if found.
 
 ## How to run this project
 
-See [this tutorial](https://karaoke-dev.github.io/how-to-install/README.html) to get ruleset from exist build.
+See [this tutorial](https://karaoke-dev.github.io/how-to-install/README.html) to get the ruleset from the existing build.
 
-Or you can compile it by yourself, `release build` then copy `osu.Game.Rulesets.Karaoke.Packed.dll` into [ruleset folder](https://github.com/LumpBloom7/sentakki/wiki/Ruleset-installation-guide)
+Or you can compile it yourself: `release build` then copy `osu.Game.Rulesets.Karaoke.Packed.dll` into your [ruleset folder](https://github.com/LumpBloom7/sentakki/wiki/Ruleset-installation-guide)
 
-If wants to try on other platform like android or ios, you might use [launcher](https://github.com/karaoke-dev/launcher).    
-Notice : The launcher is still under development, so it might not work.
+If you want to try the ruleset on another platform like android or ios, you would use [launcher](https://github.com/karaoke-dev/launcher).    
+Notice : The launcher is still under development, so it might not work as intended.
 
 ## License
 
-This repo is under [GPL V3](LICENSE) license.
-
-## Rules
-
-1. If you plan on using this repo for commercial purposes, please contact me(andy840119@gmail.com) to get permission first.
-2. Use this repo to create/use `PIRATED` karaoke beatmap is absolutely forbidden.
+This repo is covered under the [GPL V3](LICENSE) license.
+If you plan on using this repo for commercial purposes, please contact me at (andy840119@gmail.com) to get permission first.
+Using this repo to create a `PIRATED` karaoke ruleset is absolutely forbidden.
 
 ## Thanks to
 
-- [osu!](https://github.com/ppy/osu) and it's [framework](https://github.com/ppy/osu-framework), karaoke! is modified from them.
+- [osu!](https://github.com/ppy/osu) as it's [framework](https://github.com/ppy/osu-framework) for karaoke!
 
-- [RhythmKaTTE](http://juna-idler.blogspot.com/2016/05/rhythmkatte-version-01.html) and [RhythmicaLyrics](http://suwa.pupu.jp/RhythmicaLyrics.html), an open-source software to create lyric with time tag.
-Some of lyric editor in this ruleset is inspired from them.
+- [RhythmKaTTE](http://juna-idler.blogspot.com/2016/05/rhythmkatte-version-01.html) and [RhythmicaLyrics](http://suwa.pupu.jp/RhythmicaLyrics.html), an open-source software used to create lyrics with a time tag.
```

Comment:
One karaoke song has many timetags.

Reply:
That is true, but because lyrics is plural here, it implies that each lyric has its own time tag. You are right though I will change to make it clearer

Reply:
Fixed